app/locales/ht.json

```diff
@@ -1,31 +1,37 @@
 {
   "label": {
     "about_title": "Apropo",
-    "authorities_add_button_label": "Ajoute yon sous de konfyans",
+    "authorities_add_button_label": "Ajoute yon sous ou fè konfyans",
     "authorities_add_url": "Ajoute otorite a pati yon URL",
-    "authorities_desc": "Ou ka jwenn done nan men youn oswa plizyè otorite swen sante ke ou fè konfyans. Swa ou chwazi yon non nan rejis mondyal la, oswa ou antre non adrès entènèt yon otorite ou fè konfyans ki aplike chemen sekirize yo.",
+    "authorities_desc": "Chwazi otorite swen sante ou fè konfyans nan zòn ou an pou jwenn done ekspoze yo. Swa ou chwazi yon non ki soti nan rejis global la, oswa antre non adrès entènèt yon otorite, ki te aplike Chemen Sekirize, te bay.",
     "authorities_input_placeholder": "Kole URL ou an la",
-    "authorities_no_sources": "Ou poko genyen sous done! Peze bouton anba pou ou ajoute yonn.",
+    "authorities_no_sources": "Ou poko genyen sous done!",
     "authorities_removal_alert_cancel": "Anile",
     "authorities_removal_alert_desc": "Eske ou si ke ou vle retire sous done otorite sa?",
     "authorities_removal_alert_proceed": "Kontinye",
     "authorities_removal_alert_title": "Retire otorite",
     "authorities_title": "Sous ou ka fè konfyans",
+    "choose_provider_subtitle": "Pou kapab toujou rete enfòme a ekspozisyon yo, ou bezwen enskri nan yon otorite sante.",
     "choose_provider_title": "Chwazi otorite sante",
     "commitment": "Angajman",
-    "commitment_para": "Chemen sekirize yo byen anrejistre e yo tckeke entèraksyon ak moun ki itilize posisyon ou. Done ou yo pap janm kite telefòn ou san konsantman ou.",
-    "event_history_title": "Lis evènman",
-    "export_para_1": "Tanpri fè pati  pa ou nan kanpe COVID-19 lèw pataje list pozisyon ou ak otorite lokal yo.",
-    "export_para_2": "Lokalizasyon yop rale nan fòm yon seep lis lè akm kowòdone zòn yo, okenn enfòmasyon pèsonèl pap kite transfere",
+    "commitment_para": "Chemen Sekirize byen anrejistre e tckeke entèraksyon ak moun ki itilize posisyon ou. Done ou yo pap JANM kite telefòn ou san konsantman ou.",
+    "default_news_site_name": "",
```

Comment:
Please recommend translation here.

Reply:
I think "COVID Safe Paths" is fine.

Reply:
Letting it fall back to english then, thanks